test/loader.js

```diff
@@ -0,0 +1,8 @@
+describe('Handlebars.helpers.loader', function() {
+
+    it('should return a div', function() {
```

Comment:
yea, I thought about that. I was thinking it wasn't worth it because I'd have to mock so much of the logic in the other branches since they depend on things outside the repo.